filter_graph_test.go

```diff
@@ -247,6 +247,43 @@ func TestFilterGraph(t *testing.T) {
 	require.Equal(t, 1, len(os))
 	require.Equal(t, MediaTypeAudio, os[0].MediaType())
 
+	fg3 := AllocFilterGraph()
```

Comment:
I don't really see the point of adding this, it's redundant with [this](https://github.com/asticode/go-astiav/blob/master/filter_graph_test.go#L217).

Instead could you add [here](https://github.com/asticode/go-astiav/blob/master/filter_graph_test.go#L196) something like:
```go
require.Equal(t, len(buffersrcContexts)+1, fg.NbFilters())
fs := fg.Filters()
require.Equal(t, len(buffersrcContexts)+1, len(fs))
require.Equal(t, buffersinkContext.FilterContext(), fs[0])
for idx, c := range fs[1:] {
  require.Equal(t, buffersrcContexts[idx].FilterContext(), c)
}
```
I don't know whether the buffersink context comes first in `fg.Filters()` therefore you may have to reverse order when testing it.